core/backend/package.json

```diff
@@ -91,10 +91,10 @@
   },
   "dependencies": {
     "@bentley/imodeljs-native": "4.1.9",
-    "@itwin/cloud-agnostic-core": "^2.0.0",
+    "@itwin/cloud-agnostic-core": "^2.1.0",
     "@itwin/core-telemetry": "workspace:*",
-    "@itwin/object-storage-azure": "^2.0.0",
-    "@itwin/object-storage-core": "^2.0.0",
+    "@itwin/object-storage-azure": "^2.1.0",
```

Comment:
should note, this adds new peers: https://github.com/iTwin/object-storage/commit/5cd37496078bdad0431a210f2547cc8c1be9c76e